src/components/PopupExamples.js

```diff
@@ -0,0 +1,41 @@
+// TODO: Remove this file asap )
```

Comment:
Why did you add it ))?

Reply:
Read next line ))

Reply:
I see, but why did you add something in order to delete in the future ))?

Reply:
Because it is useless ))

Reply:
if it is useless so ??

https://ru.wikipedia.org/wiki/YAGNI